app/views/main.scala.html

```diff
@@ -6,7 +6,12 @@
     Currently being used for digital pack page and to help transition to a consistent
     body width throughout the site. The extra classes will be removed when this is the case.
 *@
-@(title: String, jsVars: JsVars = JsVars.default, footer: Html = Html(""), bodyClasses: Seq[String] = Nil)(content: Html)
+@(
+    title: String,
+    jsVars: JsVars = JsVars.default,
+    isInternational: Boolean = false,
```

Comment:
Not crucial, but as we keep adding new arguments to this function, it probably would be good to group related parameters into a single object.

```scala
case class PageConfig(
         title: String, 
         isInternation: Boolean,
         bodyClasses: Seq[String] = Seq.empty
) 

object PageConfig {
   def withTitle(title: String) = ???
    val default = ???
}
```

Something like this would help us keep the signature of `@main` short and simple: 

```
   @main(pageConfig: PageConfig  = PageConfig.default,
                jsVars: JsVars = JsVars.default)
```